econplayground/main/tests/test_views.py

```diff
@@ -73,7 +73,7 @@ def test_post(self):
                 'cohort_pk': self.graph.topic.cohort.pk,
                 'pk': self.graph.pk,
             }), {
-                'line1': 'up',
+                'gLine1OffsetY': '1.0',
```

Comment:
We shouldn't remove the tests which use the `line1` naming scheme here, but I am in favor of possibly adding new tests if it makes sense with the evaluation scheme.

Reply:
In the Viewer component, the React application listens for custom events triggered by the graph system, and appends these as hidden `<input>` fields to be included in the POST request: https://github.com/ccnmtl/econplayground/blob/master/media/js/src/Viewer.jsx#L98

This system uses the `line1` / `line2` up/down naming system.